CollectObject.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import awkward as ak
+
+class CollectObject:
+
+    def __init__(self, events):
+        self.events = events
+
+    def CreateMask(self, variable, cut):
+        loaded = self.objects[variable]
+        sign = cut[0]
+        value = cut[1]
+        if sign == "==":
+            self.mask = self.mask & (loaded == value)
+        elif sign == ">=":
+            self.mask = self.mask & (loaded >= value)
+        elif sign == ">":
+            self.mask = self.mask & (loaded > value)
+        elif sign == "<=":
+            self.mask = self.mask & (loaded <= value)
+        elif sign == "<":
+            self.mask = self.mask & (loaded < value)
+        elif sign == "!=":
+            self.mask = self.mask & (loaded != value)
+        else:
+            pass
+
+    def Electron(self, cuts):
+        self.objects = self.events["Electron"]
+        self.mask = ak.ones_like(self.objects["pt"], dtype=bool)
+        for variable, cut in cuts.items():
+            self.CreateMask(variable, cut)
+        return self.objects[self.mask]
```

Comment:
not implemented yet but `CollectObject` can also take the campaign as an input and we can do

```
if campaign == "RunIISummer20UL18":
    mask_hem = eta < blah & phi > blah
    self.mask = self.mask & mask_hem
```

So we can keep all object selection issues here